### PR TITLE
New version: ctrl-f.userdiag version 24.2.5

### LINKED ISSUE #136463: [Update Request]: ctrl-f.userdiag 24.1.17
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

ctrl-f.userdiag

### Package Version

24.1.18

### Please describe the changes you would like to see

New version: 24.1.18
Installer URL: https://userdiag.com/download?winget

### PR DESCRIPTION
**Pull request by Trenly**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

* Resolves #136463

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/136739)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\ctrl-f.userdiag.24.1.17\download.exe' with working directory 'D:\TOOLS'. The system cannot find the file specified.)  ---> System.ComponentModel.Win32Exception (2): An error occurred trying to start process 'C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\ctrl-f.userdiag.24.1.17\download.exe' with working directory 'D:\TOOLS'. The system cannot find the file specified.    --- End of inner exception stack trace ---    at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)

(Automated response - build 669.)

**Review by SkyEmie — APPROVED** [review]

**Comment by SkyEmie** [comment]
There is something I can do about this `Validation-Executable-Error` flag to help you?
I'm not quite familiar with the winget repo

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  2024-01-29 14:53:18.399 [CORE] DeliveryOptimization error: 0x80190193, extended error: 0x00000000 2024-01-29 14:53:18.449 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(289)\WindowsPackageManager.dll!00007FFB965AA465: (caller: 00007FFB965AAD01) Exception(1) tid(15e8) 80190193 Forbidden (403).

(Automated response - build 679.)

**Comment by stephengillie** [comment]
image

**Comment by stephengillie** [comment]
URL: https://userdiag.com/download?winget 
Status Code: 200

(Automated message - build 679)

File: manifests/c/ctrl-f/userdiag/24.1.17/ctrl-f.userdiag.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+PackageVersion: 24.1.17
+InstallerType: exe
+Scope: user
+InstallerSwitches:
+  Silent: /s
+  SilentWithProgress: /s
+Installers:
+- Architecture: neutral
+  InstallerUrl: https://userdiag.com/download?winget
+  InstallerSha256: A24E199305150F3CC981F5E23FFAE8840C21A9105D956CBF61B1AE6A50A2BBB8
```
Comment (SkyEmie) [inline manifests/c/ctrl-f/userdiag/24.1.17/ctrl-f.userdiag.installer.yaml:14]: Try like that again, i guess it should work
```suggestion
  InstallerSha256: 8E84A8E58670EFECCBEE9A11589A0E1ECD28DB2420F367FF44AAB282F52BCF0A
```

**Comment by SkyEmie** [comment]
my feedback is just above if needed

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Does the package write anything to Control Panel / Apps&Features? I'm not seeing anything in either and in the output of `winget list`

**Comment by Trenly** [comment]
> Does the package write anything to Control Panel / Apps&Features? I'm not seeing anything in either and in the output of `winget list`

It doesn't appear that it does; I do know that this package used to exist, but was removed by coolplaylinbot instead of being updated

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\ctrl-f.userdiag.24.2.5\download.exe' with working directory 'D:\TOOLS'. The system cannot find the file specified.)  ---> System.ComponentModel.Win32Exception (2): An error occurred trying to start process 'C:\Users\VALIDA~1\AppData\Local\Temp\WinGet\ctrl-f.userdiag.24.2.5\download.exe' with working directory 'D:\TOOLS'. The system cannot find the file specified.    --- End of inner exception stack trace ---

(Automated response - build 726.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**